policyengine_us/variables/gov/states/ma/tax/income/credits/ma_dependent_or_dependent_care_credit.py

```diff
@@ -13,9 +13,12 @@ class ma_dependent_or_dependent_care_credit(Variable):
     defined_for = StateCode.MA
```

Comment:
add documentation metadata stating that from 2023 this is called the Child and Family tax credit